### PR TITLE
issues #11630, #11636, #1137 and #1167

### PR DESCRIPTION
**Pull request by cyberCharl**
I think this is correct, but can't run the test since I keep getting Empty Suite error. Even though I'm following the instructions given in the Setting Up wiki regarding the Empty Suite error.

### CODE REVIEW

**Comment by cyberCharl** [comment]
I got the Empty Suite issue sorted, all 4 tests passed for moveaxis. I opened another issue (#11636) and commited it after all the tests succeeded.

**Comment by cyberCharl** [comment]
I apologize for including 4 commits of different tasks in 1 PR, I had intended to open separate PR's for the issues but GutHub pulled all my commits into the same PR.

**Review by MahmoudAshraf97 — CHANGES_REQUESTED** [review]
please remove the functions that don't work so we can merge

**Comment by MahmoudAshraf97** [comment]
if you want to create separate PRs, create a different branch for each issue

File: ivy/functional/backends/paddle/experimental/manipulation.py [hunk 1]
```diff
@@ -133,7 +139,7 @@ def dstack(
     *,
     out: Optional[paddle.Tensor] = None,
 ) -> paddle.Tensor:
-    raise IvyNotImplementedException()
+    return paddle.dtack(arrays)
```
[the remark this answers is not in the document]
Comment (cyberCharl) [reply ivy/functional/backends/paddle/experimental/manipulation.py]: removed the incorrect code for both dstack and flipud

**Pull request merged**